features/org.wso2.carbon.micro.integrator.server.feature/pom.xml

```diff
@@ -223,6 +223,10 @@
                                 <bundleDef>org.wso2.carbon:org.wso2.carbon.addressing:${carbon.kernel.version}
                                 </bundleDef>
 
+                                <!--Management API-->
+
+                                <bundleDef>org.wso2.ei:org.wso2.carbon.micro.integrator.management.apis:${product.ei.version}</bundleDef>
```

Comment:
Do we need this much new lines spaces here?

Reply:
Fixed in 1a041a80d14e7cfd3b221ddbcc9866fba9aafe0d

Reply:
It seems the spaces are back